libs/common/src/components/screens/create-wallet/index.tsx

```diff
@@ -45,6 +54,39 @@ export const CreateWalletScreen = observer<CreateWalletScreenProps>(
             Alert.alert("Something went wrong", response.payload.description);
             return;
           }
+          if (theme.loginModal) {
+            const wallet = walletsStore.currentWallet;
```

Comment:
You should probably move that code into its own function since this is already used in the existing settings screen and you also need the same behavior when login in / recovering.